src/env.saiil.yml

```diff
@@ -6,26 +6,26 @@ channels:
   - defaults
 dependencies:
   - cuda100=1.0=0
-  - cudatoolkit=10.0.130=0
+  - cudatoolkit
   - ipython
   - matplotlib
   - multidict
-  - pillow=6.1.0
-  - pip=19.2.2=py36_0
+  - pillow
+  - pip
   - protobuf=3.9.1
   - pyexcel-ods3
-  - pytorch=1.3.0
-  - tensorboard=1.14.0
-  - tensorboardx=1.8
-  - tensorflow=1.14.0
+  - pytorch
```

Comment:
why remove all of the version numbers? which ones did you remove on purpose vs. by accident?